sdk/core/azure-core/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Key Bugs Fixed
 
+- Removed `chardet` dependency in `azure.core.rest`.
```

Comment:
Do we want to clarify this is a breaking change?

Reply:
I'm not sure, @annatisch do you think we should clarify it's breaking, because this is in a provisional package? Worried we're being scary for non-azure.core.rest users for nothing